backend/tests/unit/dal/Services/PropertyServiceTest.cs

```diff
@@ -42,7 +42,7 @@ public class PropertyServiceTest
                 new object[] { new AllPropertyFilter() { PropertyType = Entity.PropertyTypes.Building, Agencies = new int[] { 3 } }, new[] { 1, 3 }, 6, 6 },
                 new object[] { new AllPropertyFilter() { PropertyType = Entity.PropertyTypes.Building, ClassificationId = 2 }, new[] { 1, 3 }, 1, 1 },
                 new object[] { new AllPropertyFilter() { PropertyType = Entity.PropertyTypes.Building, Description = "DescriptionTest" }, new[] { 1, 3 }, 1, 1 },
-                new object[] { new AllPropertyFilter() { PropertyType = Entity.PropertyTypes.Building, Municipality = "Municipality" }, new[] { 1, 3 }, 11, 10 },
+                new object[] { new AllPropertyFilter() { PropertyType = Entity.PropertyTypes.Building, Municipality = "Municipality" }, new[] { 1, 3 }, 10, 10 },
```

Comment:
I know that this is test code, but could we get a comment explaining what the expected structure of this object array is? Unless that is supposed to be obvious and I'm being thick. It just seems like there are a lot of "magic" numbers here. More specifically:

this part isn't very readable: new[] { 1, 3 }, 1, 1 }
this part of each line is readable most of the time: new AllPropertyFilter() { PropertyType = Entity.PropertyTypes.Land, ClassificationId = 2 }